client/app/api/course/Survey/Surveys.js

```diff
@@ -113,6 +113,20 @@ export default class SurveysAPI extends BaseSurveyAPI {
     return this.getClient().get(`${this._getUrlPrefix()}/${surveyId}/results`);
   }
 
+  /**
+  * Updates the ordering of questions within the survey.
+  *
+  * @param {Array.<Array>} ordering
```

Comment:
We should document the exact types, it's more expressive than text. `{Array.<{section_id: number, Array.<question_id: number>}>}`( is my understanding correct ? )